identity/src/main/java/me/uport/sdk/identity/HDAccount.kt

```diff
@@ -0,0 +1,73 @@
+package me.uport.sdk.identity
+
+import android.content.Context
+import android.support.annotation.VisibleForTesting
+import com.uport.sdk.signer.Signer
+import com.uport.sdk.signer.UportHDSigner
+import com.uport.sdk.signer.UportHDSignerImpl
+import kotlinx.serialization.Optional
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.Transient
+import kotlinx.serialization.json.Json
+
+/**
+ * This is an Account implementation for "KeyPair" account type.
+ */
+@Serializable
+data class HDAccount(
+
+        @VisibleForTesting(otherwise = VisibleForTesting.PACKAGE_PRIVATE)
+        @SerialName("uportRoot")
+        override val handle: String,
+
+        @SerialName("devKey")
+        override val deviceAddress: String,
+
+        @SerialName("network")
+        override val network: String,
+
+        @SerialName("proxy")
+        override val publicAddress: String,
+
+        @SerialName("signerType")
+        override val type: AccountType = AccountType.HDKeyPair,
+
+        @Optional
+        @SerialName("isDefault")
+        override val isDefault: Boolean? = false
+
+) : Account {
+
+    @Transient
+    val address: String
+        get() = getMnid()
```

Comment:
This should be the public address
```suggestion
        get() = publicAddress
```